repondeur/zam_repondeur/models/visionneuse.py

```diff
@@ -0,0 +1,70 @@
+import base64
+from typing import Any, List, Optional
+
+from dataclasses import dataclass, field
+
+
+@dataclass
+class Article:
+    pk: str
+    id: int
+    titre: str
+    etat: Optional[str] = ""
+    multiplicatif: Optional[str] = ""
+    jaune: Optional[str] = ""
+    alineas: Optional[dict] = None
+    amendements: Any = field(default_factory=lambda: [])  # List[Amendement]
```

Comment:
Why the convoluted type annotation here?

Reply:
Because you cannot reference `Amendement` which is declared below.

Reply:
Try putting in in double quotes, mypy should handle that.